### PR TITLE
Edit data/jira wp 12 user listing editdata

### PR DESCRIPTION
**Pull request by davidmtacc**
## Overview
Added Edit Modal which updates User's Name, Email, Status, Role and Notes
…

## Related

…

## Changes
Added edit, success, and error templates. edited table to show dropdown choice. Added Edit User function query
…

## Testing

1. Tested using test db

## UI
Screenshot 2023-07-18 at 2 15 44 PM
Screenshot 2023-07-18 at 2 15 55 PM
Screenshot 2023-07-18 at 2 16 12 PM

### CODE REVIEW

File: apcd-cms/src/apps/view_users/templates/view_user_edit_modal.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
+{% load static %}
+<link rel="stylesheet" href="{% static 'apcd-cms/css/modal.css' %}">
+<link rel="stylesheet" href="{% static 'view_users_table/css/modal.css' %}">
+<div id="editUserModal_{{r.user_id}}" class="modal fade" role="dialog">
+  <div class="modal-dialog modal-lg">
+
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title">Edit User ID: {{r.user_id}} for {{r.user_name}}</h4>
```
Comment (edmondsgarrett) [inline apcd-cms/src/apps/view_users/templates/view_user_edit_modal.html:10]:
```suggestion
        <h4 class="modal-title">Edit User: {{r.user_name}} ({{r.user_id}})</h4>
```
I had an idea for the header of the view & edit modals, let me know what you think!

File: apcd-cms/src/apps/view_users/templates/view_user_modal.html [hunk 2]
```diff
@@ -1,26 +1,30 @@
...
+<link rel="stylesheet" href="{% static 'apcd-cms/css/modal.css' %}">
+<link rel="stylesheet" href="{% static 'view_users_table/css/modal.css' %}">
+<div id="viewUserModal_{{r.user_id}}" class="modal fade" role="dialog">
   <div class="modal-dialog modal-lg">
 
     <!-- Modal content-->
     <div class="modal-content">
       <div class="modal-header">
-        <h4 class="modal-title">User Info</h4>
+        <h4 class="modal-title">Information for ID: {{r.user_id}} for {{r.user_name}}</h4>
```
Comment (edmondsgarrett) [inline apcd-cms/src/apps/view_users/templates/view_user_modal.html:10]:
```suggestion
        <h4 class="modal-title">Information for User: {{r.user_name}} ({{r.user_id}})</h4>
```

**Review by edmondsgarrett — APPROVED** [review]
Code looks good, edit action working consistently, nice work!

Had a couple of ideas for some wording.

File: apcd-cms/src/apps/view_users/templates/view_user_edit_modal.html (hunk 1 above)
Comment (davidmtacc) [reply apcd-cms/src/apps/view_users/templates/view_user_edit_modal.html:10]: yes! Sounds great.

**Review by sophia-massie — APPROVED** [review]
One suggestion to make modal titles consistent. LGTM!

File: apcd-cms/src/apps/view_users/templates/view_user_modal.html [hunk 3]
```diff
@@ -1,26 +1,30 @@
...
+<link rel="stylesheet" href="{% static 'apcd-cms/css/modal.css' %}">
+<link rel="stylesheet" href="{% static 'view_users_table/css/modal.css' %}">
+<div id="viewUserModal_{{r.user_id}}" class="modal fade" role="dialog">
   <div class="modal-dialog modal-lg">
 
     <!-- Modal content-->
     <div class="modal-content">
       <div class="modal-header">
-        <h4 class="modal-title">User Info</h4>
+        <h4 class="modal-title">Information for User: {{r.user_name}} ({{r.user_id}})</h4>
```
[the remark this answers is not in the document]
Comment (davidmtacc) [reply apcd-cms/src/apps/view_users/templates/view_user_modal.html:10]: Any and all suggestions welcome.

**Pull request merged**